core/frontend/src/tile/TileAdmin.ts

```diff
@@ -1256,6 +1259,8 @@ export namespace TileAdmin { // eslint-disable-line no-redeclare
     relaxed: 2.5 * 1024 * 1024 * 1024, // 2.5 GB
   };
 
+  export const nonMobileUndefinedGpuMemoryLimit = 6000 * 1024 * 1024; // 6,000 MB - used when nonMobile limit is undefined
```

Comment:
This is implicitly `@public`. If you didn't intend that, add `@internal`. If you did intend that, give it a better name and document it.

Reply:
Fixed to use `/** doc comment */`.
You'll need to `npm run build` core-frontend and then `npm run extract-api`.